lib/mix/test/mix/tasks/test_test.exs

```diff
@@ -297,6 +297,38 @@ defmodule Mix.Tasks.TestTest do
     end)
   end
 
+  describe "umbrella with file path" do
+    test "finds file in umbrella when root apps path specified" do
```

Comment:
Btw, we can probably merge those two tests into one. Those tests are very expensive, since they need to start a separate suite, so generally try to perform multiple assertions.

You can also just add an assertion of what happen when you run `mix(["test", "apps/unknown_app/test"])`, without creating a whole new test case, for the same reasons.

Reply:
👍 Merged the tests into one (with the individual test reasoning added as failure messages instead) and added an assertion on the unknown app call.